prow/config/jobs.go

```diff
@@ -104,6 +105,8 @@ type JobBase struct {
 	Spec *v1.PodSpec `json:"spec,omitempty"`
 	// BuildSpec is the Knative build spec used if Agent is knative-build.
 	BuildSpec *buildv1alpha1.BuildSpec `json:"build_spec,omitempty"`
+	// PipelineRunSpec is the tekton pipeline spec used if Agent is tekton-pipeline.
+	PipelineRunSpec *pipelinev1alpha1.PipelineRunSpec `json:"pipeline_run_spec,omitempty"`
```

Comment:
do we need to ensure the `PipelineRunSpec` will be mutual exclusive with `Spec` and `BuildSpec`? (I think we have a config test for that?)

Reply:
It should be in the validation on config load, which is called in the config test

Reply:
Done.